various-db-engines-it/db-it/mongodb-it/src/main/java/io/vertx/openshift/mongodb/exceptions/UnprocessableEntytyException.java

```diff
@@ -0,0 +1,7 @@
+package io.vertx.openshift.mongodb.exceptions;
+
+public class UnprocessableEntytyException extends Exception {
```

Comment:
`Entyty` :-D

Reply:
it's Unprocessable, that may explain why ;-)